src/Command/Ide/Wizard/IdeWizardCreateSshKeyCommand.php

```diff
@@ -237,12 +237,15 @@ protected function userHasUploadedIdeKeyToCloud(): bool {
    * @return \AcquiaCloudApi\Response\EnvironmentResponse|null
    * @throws \Exception
    */
-  protected function getDevEnvironment($cloud_app_uuid): ?EnvironmentResponse {
+  protected function getAnyAhEnvironment(string $cloud_app_uuid): ?EnvironmentResponse {
     $acquia_cloud_client = $this->cloudApiClientService->getClient();
     $environment_resource = new Environments($acquia_cloud_client);
     $application_environments = iterator_to_array($environment_resource->getAll($cloud_app_uuid));
     foreach ($application_environments as $environment) {
-      if (AcquiaDrupalEnvironmentDetector::isAhDevEnv($environment->name)) {
+      if (AcquiaDrupalEnvironmentDetector::isAhDevEnv($environment->name)
```

Comment:
@grasmash do we even care if it's dev/stage/prod? Should we just return the first environment found?

Reply:
That's a good point. E.g. when we have Cloud IDE for ACF they won't have a prod env unless they upgrade to ACP.

Reply:
I think that production is the only environment that has a different set of permissions, so perhaps we should only check that it’s not production.

Reply:
> perhaps we should only check that it’s not production

Agreed. Let's also throw an exception in the event no suitable environment is found, or gracefully bail on the SSH check but finish the rest of the command. Anything except a PHP error and silent failure.